docs/pages/admin-guides/access-controls/sso/sso.mdx

```diff
@@ -615,6 +615,55 @@ If something is not working, we recommend to:
 
 - Double-check the host names, tokens and TCP ports in a connector definition.
 
+## Identifier-first login
+
+Identifier-first login is a feature that simplifies the login experience in clusters with
+multiple authentication connectors. When identifier-first login is enabled, the initial
+login screen prompts the user for their username. After entering their username, the 
+user is only presented with the connector(s) mapped to them.
+
+This feature can be enabled on a cluster by configuring any one or more SAML, OIDC, or
+GitHub auth connector with a `spec.user_matchers` field, which takes a set of one or more
+user matchers for usernames. A user matcher is a string matcher that can either be a specific
+username to match, or a glob pattern using wildcard characters (`*`). An auth connector
+will be mapped to a user if one or more of its `user_matchers` matches their username.
+
+Once enabled, any users without an auth connector mapped to them will not be able to see
+any auth connectors on the login page. For this reason, it is recommended to configure
+one or more fallback connectors. A user matcher of `*` is a special case that marks the
+connector as a fallback, which will only be presented to users who don't match any other connector.
+
+#### Examples
```

Comment:
Headings below the H3 level don't render brilliantly on our docs site, so if we can, I would use regular body text to introduce each example.